sympy/calculus/tests/test_singularities.py

```diff
@@ -80,12 +80,12 @@ def test_is_strictly_decreasing():
         1/(x**2 - 3*x), Interval.Ropen(-oo, Rational(3, 2)))
     assert not is_strictly_decreasing(-x**2, Interval(-oo, 0))
     assert not is_strictly_decreasing(1)
-    assert is_strictly_decreasing(1/(x**2 - 3*x), Interval.open(1.5, 3))
```

Comment:
This test was failing on PyPy, but not in any of the other tests. It makes sense that it should fail.